clang/tools/3c/README.md

```diff
@@ -0,0 +1,35 @@
+# `3c`: Command-line tool for conversion of C code to Checked C
+
+The `3c` tool is a command-line interface to the 3C software for conversion of C code to Checked C.  See the [3C readme](../../docs/checkedc/3C/README.md) for general information about 3C and how to install it.  This document describes how to use `3c`.  It assumes that you have added the `build/bin` directory containing the `3c` executable to your `$PATH`.
+
+`3c` supports an iterative workflow for converting a C program to Checked C as follows:
+
+1. Run `3c`.  It reads a set of `.c` files and the header files they transitively include (these files may be in Checked C or, as a special case, plain C), performs a whole-program static analysis to infer as many additions or corrections to Checked C annotations as possible, and writes out the updated files.  `3c` accepts various flags to control the assumptions it makes and the types of changes it makes on a given pass; you can adjust these flags to suit your needs.
+
```

Comment:
You can drop "; you can adjust these flags to suit your needs".

Reply:
Done.